packages/outline-docs/src/guides/component-development/06-properties.mdx

```diff
@@ -21,27 +21,34 @@ import '@phase2/outline-container';
   bottom-margin="spacing-8"
 >
   ## Description
-  Lit components receive input and store their state as JavaScript class fields or properties.
+  Outline components receive input and store their state as JavaScript class fields(normally called attributes) or properties.
   
   [Reactive properties](https://lit.dev/docs/components/properties/) are properties that can trigger the reactive update cycle when changed, re-rendering the component, and optionally be read or written to attributes.
 
   ## Documentation Status
-  <outline-alert status="warning" size="large">
-    <span slot="header">Status: Needs Work/In Progress</span>
+  <outline-alert status="success" size="large">
+    <span slot="header">Status: Complete / Update to data May 2023</span>
     <p>
       This documentation is in need of additional work, or is currently in progress.
     </p>
   </outline-alert>
 </outline-container>
 
 ### Property usage basics
-
 - Properties should be commented using [JSDoc](https://github.com/jsdoc/jsdoc).
 - Each property should provide a comment including a description of the attribute functionality or purpose.
 - Pass to `type` the appropriate [converter](https://lit.dev/docs/components/properties/#conversion-type) (`String`, `Boolean`, etc.).
 - Pass to `attribute` an attribute name in kebab-case.
 - Attribute names must not collide with HTML standard ones `title`, `href`, etc.
   - This means it is best to name properties more like `link-title`, `link-href`, etc.
+- Lit [documentation]()
+
+### Slots versus properties
+- When do I use a property? 
+  - Properties are used for non-content features that impact the state of the component. For example, active versus inactive states.
+- The component shouldn't change its own public properties, except in response to user input. For example, you could set a property to change background color, but the component itself shouldn't change it's own color.
+- Slots are used for content, such as HTML markup, text, icons, or images. 
+- There are cases in which adding content as a property will make sense, for example using a title to control the state of the component. Use this cautiously, as it can cause issues with a11y tools and automation.
```

Comment:
Do we have specific examples of how this can go wrong with a11y tools? My understanding has been that that was really just a lag in the a11y tools support for web components/shadow dom, which is quickly be addressed as web component usage is exploding across the web with increased browser support.